data/core/commands/core.lua

```diff
@@ -86,6 +86,15 @@ command.add(nil, {
   end,
 
   ["core:open-file"] = function()
+    local node = core.root_view:get_active_node_default()
```

Comment:
I think that we can do it more easily using `core.active_view`

Reply:
Good call! Fixed.